app/src/renderer/connectors/lcdClient.js

```diff
@@ -90,12 +90,60 @@ Object.assign(Client.prototype, {
   },
   tx: argReq("GET", "/txs"),
 
-  // staking
-  updateDelegations: req("POST", "/stake/delegations"),
+  /* ============ STAKE ============ */
+
+  // Get all delegations information from a delegator
+  getDelegator: function(addr) {
+    return req("GET", `/stake/delegators/${addr}`).call(this)
+  },
+  // Get all txs from a delegator
+  getDelegatorTxs: function(addr) {
+    return req("GET", `/stake/delegators/${addr}/txs`).call(this)
+  },
+  // Get a specific tx from a delegator
+  getDelegatorTx: function(addr, id, types) {
+    if types ===  "" {
+      return req("GET", `/stake/delegators/${addr}/txs`).call(this)
+    } else {
+      return req("GET", `/stake/delegators/${addr}/txs?type=${types}`).call(this)
+    }
+  },
+  // // Query all validators that a delegator is bonded to
+  // getDelegatorValidators: function(delegatorAddr) {
+  //   return req("GET", `/stake/delegators/${delegatorAddr}/validators`).call(this)
+  // },
+  // // Query a validator info that a delegator is bonded to
+  // getDelegatorValidator: function(delegatorAddr, validatorAddr) {
+  //   return req("GET", `/stake/delegators/${delegatorAddr}/validators/${validatorAddr}`).call(this)
+  // },
+
+  // Get a list containing all the validator candidates
+  getValidators: req("GET", "/stakes/validators/"),
```

Comment:
typo

Reply:
for some reason my ESlint wasn't working. I had to delete it and reinstall